assets/html/scss/documenter/_elements.scss

```
@@ -33,3 +33,35 @@ h1, h2, h3, h4, h5, h6 {
     display: none !important;
   }
 }
+
+pre {
+  position: relative;
+
+  .copy-button {
+    opacity: 0;
```

Perhaps make it visible without focus too? Something like
```suggestion
    opacity: 0.2;
```
looks pretty good IMO. Thoughts?

---

It does, yeah :)